@app/db/.gmrc

```diff
@@ -68,7 +68,7 @@
     "!afterReset.sql",
     {
       "_": "command",
-      "command": "DATABASE_URL=\"${GM_DBURL}\" yarn workspace @app/worker install-db-schema"
+      "command": "node scripts/after-reset.js"
```

Comment:
I'm not keen to add a JS script that then just forks out to another command line; it's a bit of a pain to maintain and the indirection makes it harder for the user to follow. Is it possible to accomplish this with `cross-env`?

```suggestion
      "command": "cross-env-shell \"DATABASE_URL=\\\"$GM_DBURL\\\" yarn workspace @app/worker install-db-schema\""
```